go.mod

```diff
@@ -7,30 +7,33 @@ require (
 	github.com/lib/pq v1.10.6
 	github.com/sirupsen/logrus v1.9.0
 	github.com/spf13/cobra v1.5.0
+	github.com/stretchr/testify v1.8.0
```

Comment:
Nice, the code looks great! Can you run `go mod tidy` locally and push the changes to undo adding the testify package?